R/comp_corr.R

```diff
@@ -1,7 +1,7 @@
 #' Computing corr
 #'
-#' This function computes the weighted correlation with penalty (no penalty if not lagged) after the lags
-#' applied to the dataset and timepoints using prep.data() and best.lag() function.
+#' This function computes the weighted correlation with penalty (no penalty if not lagged) after the lags are applied
```

Comment:
Per @leungi's suggestion we could change this to say:
> This function computes the weighted correlation with a penalty for lags.  It should only be used after the fixed lags have already been applied to the dataset and timepoints using the functions prep.data() and best.lag().

Reply:
Done.